azafea/event_processors/endless/country.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2020 - Endless
+#
+# This file is part of Azafea
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+COUNTRIES = {
+    'ABW': 'AW', 'AGO': 'AO', 'ALA': 'AX', 'AND': 'AD', 'ARE': 'AE', 'ARM': 'AM', 'ATA': 'AQ',
+    'ATF': 'TF', 'ATG': 'AG', 'AUT': 'AT', 'BDI': 'BI', 'BEN': 'BJ', 'BES': 'BQ', 'BGD': 'BD',
+    'BHS': 'BS', 'BIH': 'BA', 'BLR': 'BY', 'BLZ': 'BZ', 'BRB': 'BB', 'BRN': 'BN', 'CAF': 'CF',
+    'CHL': 'CL', 'CHN': 'CN', 'COD': 'CD', 'COG': 'CG', 'COK': 'CK', 'COM': 'KM', 'CPV': 'CV',
+    'CUW': 'CW', 'CYM': 'KY', 'DNK': 'DK', 'ESH': 'EH', 'EST': 'EE', 'FLK': 'FK', 'FRO': 'FO',
+    'FSM': 'FM', 'GIN': 'GN', 'GLP': 'GP', 'GNB': 'GW', 'GNQ': 'GQ', 'GRD': 'GD', 'GRL': 'GL',
+    'GUF': 'GF', 'GUY': 'GY', 'IRL': 'IE', 'IRQ': 'IQ', 'ISR': 'IL', 'JAM': 'JM', 'KAZ': 'KZ',
+    'KOR': 'KR', 'LBR': 'LR', 'LBY': 'LY', 'MAC': 'MO', 'MAF': 'MF', 'MDG': 'MG', 'MDV': 'MV',
+    'MEX': 'MX', 'MLT': 'MT', 'MNE': 'ME', 'MNP': 'MP', 'MOZ': 'MZ', 'MTQ': 'MQ', 'MYT': 'YT',
+    'NIU': 'NU', 'PAK': 'PK', 'PCN': 'PN', 'PLW': 'PW', 'PNG': 'PG', 'POL': 'PL', 'PRK': 'KP',
+    'PRT': 'PT', 'PRY': 'PY', 'PYF': 'PF', 'SEN': 'SN', 'SGS': 'GS', 'SLB': 'SB', 'SLV': 'SV',
+    'SPM': 'PM', 'SRB': 'RS', 'SUR': 'SR', 'SVK': 'SK', 'SVN': 'SI', 'SWE': 'SE', 'SWZ': 'SZ',
+    'SYC': 'SC', 'TCD': 'TD', 'TKM': 'TM', 'TUN': 'TN', 'TUR': 'TR', 'TUV': 'TV', 'UKR': 'UA',
+    'URY': 'UY', 'WLF': 'WF'}
```

Comment:
It's rare but countries do come into existence and disappear again. Can we not use a library which exposes this data, rather than hardcoding a list ourselves? There is an `iso-codes` package, and https://pypi.org/project/pycountry/ seems a reasonably well-maintained wrapper around it.

Reply:
We could use an external package, but this code will be removed in the second step of the migration (when we’ll only have 2-letter codes everywhere), so there’s no need to take care of new countries appearing. Moreover, we’ll need a manual workaround for Kosovo that’s not included in the `iso-codes` package. I don’t think that relying on a Python package that needs a Debian package is really useful for some code that will be removed as soon as endlessm/eos-activation-server#23 is on production.

Reply:
OK, I'd missed that this is a temporary mapping.